discord/state.py

```diff
@@ -928,8 +928,7 @@ def parse_guild_ban_add(self, data):
 
     def parse_guild_ban_remove(self, data):
         guild = self._get_guild(int(data['guild_id']))
-        if guild is not None:
-            if 'user' in data:
+        if guild is not None and 'user' in data:
                 user = self.store_user(data['user'])
                 self.dispatch('member_unban', guild, user)
```

Comment:
Indentation error here for some reason
```suggestion
            user = self.store_user(data['user'])
            self.dispatch('member_unban', guild, user)
```